x/bridge/keeper/assets_unlocked.go

```diff
@@ -189,3 +189,34 @@ func (k Keeper) BurnERC20(
 
 	return nil
 }
+
+func (k Keeper) GetMinBridgeOutAmount(ctx sdk.Context, mezoToken []byte) math.Int {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.GetMinBridgeOutAmountKey(mezoToken))
+	if len(bz) == 0 {
+		return math.ZeroInt()
+	}
+
+	var minAmount math.Int
+	err := minAmount.Unmarshal(bz)
+	if err != nil {
+		panic(err)
+	}
+
+	return minAmount
+}
+
+func (k Keeper) SetMinBridgeOutAmount(
+	ctx sdk.Context,
+	mezoToken []byte,
+	minAmount math.Int,
+) error {
+	store := ctx.KVStore(k.storeKey)
+	bz, err := minAmount.Marshal()
+	if err != nil {
+		return err
```

Comment:
Should we panic here? pretty sure other setters in keepers panics, in any case, not being able to set this value, should be an unrecoverable error I suppose.

Reply:
Done in 02efc8f683284c0c9c078c018f10767e7d7f257a.